spec/unit/gameSpec.js

```diff
@@ -0,0 +1,86 @@
+describe("Game", function () {
```

Comment:
A number of problems in this file, with some good attempts. I'm not commenting everywhere, but try to think about whether my comments apply to different tests or not. Happy to chat to clarify if needed.